package.json

```diff
@@ -115,14 +116,15 @@
     "start-docs": "npm run build-min && npm run build-docs && jekyll serve --watch",
     "lint": "eslint --cache --ignore-path .gitignore src test bench docs/_posts/examples/*.html debug/*.html",
     "lint-docs": "documentation lint src/index.js",
+    "lint-css": "stylelint 'dist/mapbox-gl.css' || true",
```

Comment:
can we remove the `|| true` statements from here and the `test-flow` command – I'm afraid they will prevent the tests from failing if there are errors.